8.1/Dockerfile.rhel8

```diff
@@ -0,0 +1,81 @@
+FROM ubi8/s2i-base:1
+
+# This image provides an Apache+PHP environment for running PHP
+# applications.
+
+EXPOSE 8080
+EXPOSE 8443
+
+# Description
+# This image provides an Apache 2.4 + PHP 8.0 environment for running PHP applications.
+# Exposed ports:
+# * 8080 - alternative port for http
+
+ENV PHP_VERSION=8.0 \
```

Comment:
This should be as well version `8.1` and not `8.0`

Reply:
Similarly to c9s, we are not building 8.1 for RHEL8.

Reply:
Then we should either remove it or correct it. It is misleading.